core-java-modules/core-java-numbers-7/src/test/java/com/baeldung/securerandompositivelong/SecureRandomPositiveLongUnitTest.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.securerandompositivelong;
+
+import org.junit.jupiter.api.Test;
+
+import java.security.SecureRandom;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SecureRandomPositiveLongUnitTest {
+
+    @Test
+    void whenGenerateRecureRandom_thenGetExpectedValue() {
+        SecureRandom secureRandom = new SecureRandom();
+        long randomPositiveLong = Math.abs(secureRandom.nextLong());
+
+        assertThat(randomPositiveLong).isPositive();
```

Comment:
I think this could be zero too, so the assertion should be isPositiveOrZero

Reply:
Hi @theangrydev 
isPositiveOrZero() is in which version of JUnit? we are still on 5.9.2 and I can't find it.
Or may we use isGreaterThanOrEqualTo(0L) ?